oecophylla/taxonomy/parser.py

```diff
@@ -39,3 +42,36 @@ def combine_bracken(bracken_outputs):
 
     # return a merged pd.DataFrame of all absolute estimated assigned reads
     return pd.concat(samples, axis=1).fillna(0).astype(int)
+
+
+def pandas2biom(file_biom, table, err=sys.stderr):
+    """ Writes a Pandas.DataFrame into a biom file.
+
+    Parameters
+    ----------
+    file_biom: str
+        The filename of the BIOM file to be created.
+    table: a Pandas.DataFrame
+        The table that should be written as BIOM.
+    err : StringIO
+        Stream onto which errors / warnings should be printed.
+        Default is sys.stderr
+
+    Raises
+    ------
+    IOError
+        If file_biom cannot be written.
+
+    Returns
+    -------
+    Nothing
+    """
+    try:
+        bt = biom.Table(table.values,
```

Comment:
can the load into the `Table` be performed outside of the try / except? If not, would it be possible to update the error message to indicate the error was either on load or write?

Reply:
what about removing the try except completely and expose the original error to the user of the function?